packages/at_backupkey_flutter/example/ios/Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-platform :ios, '13.0'
+platform :ios, '12.0'
```

Comment:
nit: we have an N-2 policy for ios. Latest is 18.* so this should go up to 16.0 not down from 13 to 12.